Tasks/AzureVmssDeployment/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "AzureVmssDeployment",
   "main":"main.js",
   "dependencies": {
-    "vsts-task-lib" : "^2.0.0"
+    "vsts-task-lib" : "^2.0.0",
+    "artifact-mover": "1.0.11"
```

Comment:
this is temporary. Will replace it with official npm package released from vsts-rm-extensions repo

Reply:
fixed